front-end/src/app/reports/submission-workflow/submit-report-step2.component.html

```diff
@@ -95,7 +95,13 @@ <h3>Enter password</h3>
             <label for="filingPassword">
               <strong>ELECTRONIC FILING PASSWORD </strong>
             </label>
-            <input type="text" pInputText id="filingPassword" formControlName="filingPassword" />
+            <p-password
+              id="filingPassword"
+              type="password"
```

Comment:
So i tried this a while back and ran into a problem.  Because it's a password input, the browser throws in your password for the website which probably isn't the same as the filing password.  Do you think there's a way we make it a password input but disable the autofill?

Reply:
So I've been looking into this and there is an autocomplete field which you should technically be able to set to "off" to prevent this. However, modern browsers seem to have chosen to ignore it. 
https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#the_autocomplete_attribute_and_login_fields

Reply:
So after a lot of digging around I eventually found:
https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#the_autocomplete_attribute_and_login_fields

Which was a little disheartening since it seemed modern browser just wouldn't allow you to disable until you get to the very bottom and it tells you you can use:
autocomplete="new-password"

That finally did the trick for me.